### DISCUSSION TITLE
"Styling" von Textelementen und Abständen (z.B. Tab Bar Höhe, Widget und Modul Text, etc.)

### DISCUSSION THREAD

**Discussion by gesaeuse** [Zefau/ioBroker.jarvis#1420] [Q&A] [answered]
Hallo zusammen,
wie kann ich in V3 die Höhe der Tab Bar und die Textgröße + Abstände von Inhaltselementen z.B. Widget und Modul Text ändern? Habe mir mal mit Custom CSS Styles (Pro) wie folgt geholfen, aber bin mir nicht sicher, ob das aktuell der effizienteste und eleganteste Weg ist. Danke für alle Tipps!

```
/* Tab Bar Label Textgröße und Höhe */
.q-tab__label {
	font-size: 18px;
	margin-top: 25px;
	margin-bottom: 5px;
}

/* Content Bereich Abstand oben */
.jarvis-page-container {
	margin-top: 40px;
}

/* Widget Label Textgröße */
.q-item__label.text-h6 {
	font-size: 22px;
}

/* Modul Label Abstand unten */
.q-item__label--caption {
	padding-bottom: 10px;
}

/* Modul Label Textgröße */
.q-item__label {
	font-size: 16px;
}
```

**Comment by gesaeuse** [comment, marked as answer]
...habe hier alles gefunden, was ich gesucht habe:
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/styles/jarvis-klassen
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/styles/tabs/tab-zeile
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/styles/tabs/tab-icons

Hat sich somit erledigt ;-)

**Reply by Zefau** [reply]
Klasse, danke für das Feedback!
